src/modes/simple_select.js

```diff
@@ -233,16 +234,82 @@ module.exports = function(ctx, options = {}) {
       }
     },
     render: function(geojson, push) {
-      geojson.properties.active = (ctx.store.isSelected(geojson.properties.id) )
+      geojson.properties.active = (ctx.store.isSelected(geojson.properties.id))
         ? Constants.activeStates.ACTIVE
         : Constants.activeStates.INACTIVE;
       push(geojson);
       if (geojson.properties.active !== Constants.activeStates.ACTIVE
         || geojson.geometry.type === Constants.geojsonTypes.POINT) return;
       createSupplementaryPoints(geojson).forEach(push);
     },
-    trash() {
+    trash: function() {
       ctx.store.delete(ctx.store.getSelectedIds());
+    },
+    combineFeatures: function() {
```

Comment:
Yeah, we should noop internally. Studio should dim the button and show "cannot merge features with different geometry types" in a popover.